tests/test_client.py

```diff
@@ -507,6 +551,26 @@ def update_locations(self, obj_id):
             raise RuntimeError(resp.errors)
         return resp
 
+    def update_shift(self, obj):
+        body = {
+            "shift": { # TODO: Can this be obj with the title updated?
+                "employee_id": obj['employee_id'],
+                "location_id": obj['location_id'],
+                "start_at": obj['start_at'],
+                "end_at": obj['end_at'],
+                "wage": {
+                    "title": self.make_id('shift'),
+                    "hourly_rate" : obj['wage']['hourly_rate']
+                }
+            }
+        }
+        resp = self._client.labor.update_shift(id=obj['id'], body=body)
+
+        if resp.is_error():
+            raise RuntimeError(resp.errors)
+        LOGGER.info('Created a Shift with id %s', resp.body.get('shift',{}).get('id'))
```

Comment:
Here the logger is after the resp.is_error() check, but in the create method it's before it. Can we be consistent (I slightly prefer it after).

Reply:
I moved the log line in `create_shift()` and updated this one to not say `"Created a Shift... "`